integration-templates/zoho-crm/zoho-crm-accounts.ts

```diff
@@ -25,10 +25,9 @@ export default async function fetchData(nango: NangoSync) {
             await nango.batchSave(mappedAccounts, 'ZohoCRMAccount');
         }
     } catch (error: any) {
-        if (error.status = 304) {
+        if ((error.status = 304)) {
```

Comment:
shouldn't it be `if(error.status == 304)` instead. Same comment below as well

Reply:
Yeah, it should be. My IDE simply auto corrected this sync that was added by an outside contributor but worth fixing regardless